### PR TITLE
Updated description of Amazon Inspector

### PR DESCRIPTION
**Pull request by sotoiwa**
*Issue #, if available:*
#221

*Description of changes:*
The description was regarding Inspector Classic, so I have updated the description to use v2. Updated link to Inspector documentation from Classic to v2.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/security/docs/hosts.md [hunk 1]
```diff
@@ -88,7 +88,9 @@ to access the node.
...
+You can use [Amazon Inspector](https://docs.aws.amazon.com/inspector/latest/user/what-is-inspector.html) to check for unintended network accessibility of your nodes and for vulnerabilities on those Amazon EC2 instances.
+
+Amazon Inspector can provide common vulnerabilities and exposures (CVE) data for your Amazon EC2 instances only if the Amazon EC2 Systems Manager (SSM) agent is installed and enabled. This agent is preinstalled on [many Amazon EC2 instances](https://docs.aws.amazon.com/ja_jp/systems-manager/latest/userguide/ami-preinstalled-agent.html) including [EKS optimized Amazon Linux AMIs](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html). Regardless of SSM agent status, all of your Amazon EC2 instances are scanned for network reachability issues. For more information about configuring scans for Amazon EC2, see [Scanning Amazon EC2 instances](https://docs.aws.amazon.com/inspector/latest/user/enable-disable-scanning-ec2.html).
```
Comment (jimmyraywv) [inline content/security/docs/hosts.md:93]: Shouldn't we use this EN link: https://docs.aws.amazon.com/systems-manager/latest/userguide/ami-preinstalled-agent.html

Comment (jimmyraywv) [inline content/security/docs/hosts.md:93]: Consider changing:

```
...unintended network accessibility of your nodes and for vulnerabilities on those Amazon EC2 instances.
```

to

```
...unintended network access to your nodes and for vulnerabilities on the underlying Amazon EC2 instances.
```

Consider changing:
```
This agent is preinstalled on [many Amazon EC2 instances]...
```

to

```
This agent is preinstalled on several [Amazon Machine Images (AMIs)]
```

Comment (sotoiwa) [reply content/security/docs/hosts.md]: Thanks. Changed URL to EN document.

Comment (sotoiwa) [reply content/security/docs/hosts.md]: Thanks. I have revised the text as you suggested.

**Review by jimmyraywv — APPROVED** [review]
@jicowan LGTM

**Pull request merged**